tests/__init__.py

```diff
@@ -667,7 +667,7 @@ def capture(self):
     "active": True,
     "aggregate_usage": "sum",
     "amount": 200,
-    "billing_scheme": "per_unit",
```

Comment:
were these JSON file changes manual, or have those fields been removed from the API now?  I'd rather keep the fixtures matching what the API returns so we can keep updating them automatically, even if we no longer use the fields (though I realize that updating them does require manually picking which changes are relevant, like discarding unrelated id changes, which is annoying).

Reply:
(resolving this one since I commented in the wrong place)